src/xrpld/rpc/handlers/LedgerEntry.cpp

```diff
@@ -67,6 +67,43 @@ parseAuthorizeCredentials(Json::Value const& jv)
     return arr;
 }
 
+static std::optional<uint256>
+parseAccountPermission(Json::Value const& params, Json::Value& jvResult)
+{
+    // LCOV_EXCL_START
+    if (!params.isObject())
+    {
+        uint256 uNodeIndex;
+        if (!uNodeIndex.parseHex(params.asString()))
+        {
+            jvResult[jss::error] = "malformedRequest";
+            return std::nullopt;
+        }
+        return uNodeIndex;
+    }
+    // LCOV_EXCL_STOP
+    if (!params.isMember(jss::account) || !params.isMember(jss::authorize))
+    {
+        jvResult[jss::error] = "malformedRequest";
+        return std::nullopt;
+    }
+    auto const account =
+        parseBase58<AccountID>(params[jss::account].asString());
+    if (!account)
+    {
+        jvResult[jss::error] = "malformedAccount";
+        return std::nullopt;
+    }
+    auto const authorize =
+        parseBase58<AccountID>(params[jss::authorize].asString());
+    if (!authorize)
+    {
+        jvResult[jss::error] = "malformedAuthorize";
+        return std::nullopt;
+    }
```

Comment:
should double check with clio team for these error msgs. sometimes they may have trouble supporting new error msgs so we should avoid adding new ones if not necessary. the safe approach is to just return  `malformedAddress`